### PR TITLE
Show an image placeholder for the map block

### PR DESCRIPTION
**Pull request by gravityrail**
Master issue #13510

#### Changes proposed in this Pull Request:

* Added an example to the Map block. This is done using an image so that we don't need to use an API key.

#### Testing instructions:
* Create a new post
* Click the + to insert a new block
* Hover over Map

map-preview

### CODE REVIEW

File: extensions/blocks/map/edit.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ import Locations from './locations';
 import Map from './component.js';
 import MapThemePicker from './map-theme-picker';
 import { settings } from './settings.js';
+import previewPlaceholder from './map-preview.png';
```
Comment (jeherve) [inline extensions/blocks/map/edit.js]: We'll most likely need to switch to an optimized jpg file, to match the other block example images.

File: extensions/blocks/map/edit.js [hunk 2]
```diff
@@ -269,12 +279,18 @@ class MapEdit extends Component {
 				</div>
 			</Fragment>
 		);
+		const placholderPreview = (
+			<div>
+				<img alt="Map Preview" src={ previewPlaceholder } />
```
Comment (jeherve) [inline extensions/blocks/map/edit.js]: I think it would be nice if this were translated.

**Review by jeherve — CHANGES_REQUESTED** [review]

File: extensions/blocks/map/edit.js (hunk 2 above)
Comment (gravityrail) [reply extensions/blocks/map/edit.js]: Fixed

File: extensions/blocks/map/edit.js (hunk 1 above)
Comment (gravityrail) [reply extensions/blocks/map/edit.js]: Fixed - now a JPG at 512 pixel resolution

**Review by jeherve — APPROVED** [review]
This works well in my tests. Merging.

**Pull request merged**